From 342b99bc3cb4e0a7bb3a5244b848cbac83920dad Mon Sep 17 00:00:00 2001
From: Sean Cross <xobs@kosagi.com>
Date: Thu, 19 May 2016 12:49:49 +0800
Subject: [PATCH 64/65] novena_defconfig: enable CONFIG_RTC_INVALID_2038

This is in an effort to fix the systemd boot bug.

Signed-off-by: Sean Cross <xobs@kosagi.com>
---
 arch/arm/configs/novena_defconfig | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/arch/arm/configs/novena_defconfig b/arch/arm/configs/novena_defconfig
index 542f40e..6937535 100644
--- a/arch/arm/configs/novena_defconfig
+++ b/arch/arm/configs/novena_defconfig
@@ -1,6 +1,6 @@
 #
 # Automatically generated file; DO NOT EDIT.
-# Linux/arm 4.4.0-rc7 Kernel Configuration
+# Linux/arm 4.4.0 Kernel Configuration
 #
 CONFIG_ARM=y
 CONFIG_ARM_HAS_SG_CHAIN=y
@@ -3216,6 +3216,8 @@ CONFIG_DRM_IMX_FB_HELPER=y
 CONFIG_DRM_IMX_LDB=y
 CONFIG_DRM_IMX_IPUV3=y
 CONFIG_DRM_IMX_HDMI=y
+CONFIG_DRM_ETNAVIV=y
+# CONFIG_DRM_ETNAVIV_REGISTER_LOGGING is not set
 
 #
 # Frame buffer Devices
@@ -3975,6 +3977,7 @@ CONFIG_RTC_HCTOSYS_DEVICE="rtc0"
 CONFIG_RTC_SYSTOHC=y
 CONFIG_RTC_SYSTOHC_DEVICE="rtc0"
 # CONFIG_RTC_DEBUG is not set
+CONFIG_RTC_INVALID_2038=y
 
 #
 # RTC interfaces
@@ -4270,8 +4273,6 @@ CONFIG_LTE_GDM724X=m
 # CONFIG_FB_TFT is not set
 # CONFIG_WILC1000_DRIVER is not set
 # CONFIG_MOST is not set
-CONFIG_DRM_ETNAVIV=y
-# CONFIG_DRM_ETNAVIV_REGISTER_LOGGING is not set
 # CONFIG_CHROME_PLATFORMS is not set
 CONFIG_CLKDEV_LOOKUP=y
 CONFIG_HAVE_CLK_PREPARE=y
-- 
2.7.3

